feat: support `IN` operator for non-pull queries (#6475)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/operators.md

  EMIT CHANGES;
```

IN
--

> The IN operator allows you to specify multiple values in a `WHERE` clause.

Reviewer: ```suggestion
The IN operator enables specifying multiple values in a `WHERE` clause.

```

> The IN operator is a shorthand for multiple `OR` conditions.

Reviewer: ```suggestion
It provides the equivalent of multiple `OR` conditions.

```
